src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala

```diff
@@ -70,6 +73,19 @@ class WorkspaceService(protected val argUserToken: WithAccessToken, val rawlsDAO
 
   }
 
+  def getStorageCostEstimate(workspaceNamespace: String, workspaceName: String) = {
+    Future.sequence(Seq(
```

Comment:
Yes, that makes sense. I didn't like having to explicitly specify the types. I guess maybe `Future.sequence` works better when all the futures are of the same type?